backend/src/main/resources/db/migration/V0.15__announcement_norm_one_to_many.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE announcements
+    DROP CONSTRAINT announcements_eli_fkey;
```

Comment:
todo: change the existing data in the eli column from manifestation to expression eli